sway-lib-std/src/assert.sw

```diff
@@ -3,7 +3,7 @@ library assert;
 use ::logging::log;
 use ::revert::revert;
 
-const FAILED_REQUIRE_SIGNAL = 42;
+pub const FAILED_REQUIRE_SIGNAL = 42;
```

Comment:
This shouldn't really need the `pub` keyword but without it I get an IR error. Will create an issue.